caffe2/python/operator_test/pack_ops_test.py

```diff
@@ -49,6 +49,93 @@ def pack_segments_ref(lengths, data):
 
         return pack_segments_ref
 
+    def pack_segments_with_max_length_ref(self, return_presence_mask=False):
```

Comment:
could you extend `pack_segments_ref` to take an additional optional max_length argument?

Reply:
You mean delete this function and modify the existing one?

Reply:
yep